web/modules/custom/dept_migrate/modules/dept_migrate_group_files/src/Plugin/migrate/process/VideoUrl.php

```diff
@@ -25,24 +25,7 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     $url = preg_replace('|^oembed:\/\/|', '', $value);
     // Remove HTML special chars to give a user readable string.
     $url = urldecode($url);
-    // See if thumbnail is available.
-    $thumbnail_url = '';
 
-    if (preg_match('|vimeo|', $url)) {
```

Comment:
This doesn't help us here. If a thumbnail's missing, we can live with it. But we can't live without pulling the entity across into D9